src/pywink/api.py

```diff
@@ -344,6 +349,15 @@ def __get_subsensors_from_smoke_detector(item, api_interface, filter_key):
     return subsensors
 
 
+def __get_subsensors_from_door_bell(item, api_interface, filter_key):
+    if filter_key != 'door_bell_id':
+        return []
+    subsensors = []
+    subsensors.append(WinkDoorBellMotion(item, api_interface))
```

Comment:
Please add checks for the capability like in `__get_subsensors_from_sensor_pod`
This will blow up if the door bell doesn't have a motion sensor.